### PR TITLE
Fix afaanoromoo liveradio test e2e

### LINKED ISSUE #3976: Live Radio page for Afaan Oromoo has embedded recursive page displayed
**Describe the bug**
The Live Radio page for Afaan Oromoo is currently showing an embedded version of the entire live radio page instead of the audio player.

**To Reproduce**
1. Navigate to https://www.test.bbc.com/afaanoromoo/bbc_afaanoromoo_radio/liveradio
(issue is present on stage too)
2. See screenshot below
image

**Expected behavior**
The audio player controls should be displayed

**Testing notes**

- [ ] This bug fix is expected to need manual testing.
- [ ] *Note* Automated e2e tests must be re-enabled

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #3976

**Overall change:** Overrides the externalId to `bbc_afaanoromoo_radio` when it is `bbc_oromo_radio`

**Code changes:**

- Adds a helper that can override the externalId if it equals `bbc_oromo_radio`
- Adds test for the helper and snapshots of the component showing the externalId is overridden 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

This can be tested by visiting http://localhost.bbc.com:7080/afaanoromoo/bbc_afaanoromoo_radio/liveradio and seeing real AV playback

image

### CODE REVIEW

**Comment by pjlee11** [comment]
I've tested this locally by copying the `checkExternalIdOverrides` logic into the PR https://github.com/bbc/simorgh/pull/3939 and seeing valid playback on local. 

image

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/MediaPageBlocks/Blocks/LiveRadio/helper.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import checkExternalIdOverrides from './helper';
+
+describe('LiveRadio helper override externalId', () => {
+  it('should override bbc_oromo_radio to bbc_afaanormoo_radio', () => {
+    const externalId = 'bbc_oromo_radio';
+    expect(checkExternalIdOverrides(externalId)).toBe('bbc_afaanoromoo_radio');
+  });
+
+  it('should not override anything other than bbc_afaanoromoo_radio', () => {
+    let externalId = 'foobar';
```
Comment (ryanmccombe) [inline src/app/containers/MediaPageBlocks/Blocks/LiveRadio/helper.test.js]: Might be cleaner:
```
['foobar', 'bbc_korean_radio', 'do_not_override_me'].forEach(externalId => {
  expect(checkExternalIdOverrides(externalId)).toBe(externalId);
}}
```

**Review by ibMadbouly — APPROVED** [review]

File: src/app/containers/MediaPageBlocks/Blocks/LiveRadio/helper.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const checkExternalIdOverrides = externalId =>
+  externalId === 'bbc_oromo_radio' ? `bbc_afaanoromoo_radio` : externalId;
```
Comment (dr3) [inline src/app/containers/MediaPageBlocks/Blocks/LiveRadio/helper.js]: This should be in the service config for `afaanoromoo` and used if set

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/containers/MediaPageBlocks/Blocks/LiveRadio/helper.js (hunk 2 above)
Comment (pjlee11) [reply src/app/containers/MediaPageBlocks/Blocks/LiveRadio/helper.js]: IMO overkill...

I'll do it though

File: src/app/lib/config/services/afaanoromoo.js [hunk 3]
```diff
@@ -192,6 +192,9 @@ export const service = {
       },
     ],
     timezone: 'Africa/Addis_Ababa',
+    liveRadio: {
+      externalIdOverrides: [{ bbc_oromo_radio: 'bbc_afaanoromoo_radio' }],
```
Comment (dr3) [inline src/app/lib/config/services/afaanoromoo.js]: Why is this an array?

**Review by dr3 — APPROVED** [review]

**Review by chris-hinds — CHANGES_REQUESTED** [review]
~Unless I have missed it could this PR please re-enable the e2e tests on test that were disabled this morning. https://github.com/bbc/simorgh/pull/3975~

never mind I have just seen https://github.com/bbc/simorgh/pull/3987

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**